### DISCUSSION TITLE
Scan Conda dependencies

### DISCUSSION THREAD

**Discussion by tkaden** [aquasecurity/trivy#3712] [Q&A] [answered]
I noticed recent support for including conda packages in SBOMs:
- https://github.com/aquasecurity/trivy/pull/3379
- https://github.com/aquasecurity/trivy/issues/3321

But I was curious if those packages are being scanned for vulnerabilities or if that's a future feature?

**Comment by knqyf263** [comment, marked as answer]
Vulnerability scanning is not supported now.
https://aquasecurity.github.io/trivy/v0.37/docs/sbom/#supported-packages

**Reply by tkaden** [reply]
Cool. Thanks for the clarification.
